libraries/AP_RCProtocol/AP_RCProtocol_CRSF.cpp

```diff
@@ -26,6 +26,9 @@
 #include <AP_Vehicle/AP_Vehicle_Type.h>
 #include <AP_RCTelemetry/AP_CRSF_Telem.h>
 #include <AP_SerialManager/AP_SerialManager.h>
+#ifndef IOMCU_FW
```

Comment:
I don't quite understand why this is disabled for IOMCU boards?

Reply:
building iofirmware would fail, perhaps there's a better way?